### PR TITLE
Fold XRVisibilityState into XRSession.visibilityState

### PR DESCRIPTION
**Pull request by Elchi3**
Fold https://developer.mozilla.org/en-US/docs/Web/API/XRVisibilityState into https://developer.mozilla.org/en-US/docs/Web/API/XRSession/visibilityState

### CODE REVIEW

File: files/en-us/web/api/xrsession/index.html [hunk 1]
```diff
@@ -25,19 +25,13 @@ <h2 id="Properties">Properties</h2>
...
  <dd>Returns a list of this session's {{DOMxRef("XRInputSource")}}s, each representing an input device used to control the camera and/or scene.</dd>
  <dt>{{DOMxRef("XRSession.renderState", "renderState")}} {{Experimental_Inline}}{{ReadOnlyInline}}</dt>
  <dd>An {{domxref("XRRenderState")}} object which contains options affecting how the imagery is rendered. This includes things such as the near and far clipping planes (distances defining how close and how far away objects can be and still get rendered), as well as field of view information.</dd>
  <dt>{{DOMxRef("XRSession.visibilityState", "visibilityState")}} {{Experimental_Inline}}{{ReadOnlyInline}}</dt>
- <dd>A {{domxref("DOMString")}} whose value is one of those found in the {{domxref("XRVisibilityState")}} enumerated type, indicating whether or not the session's imagery is visible to the user, and if so, if it's being visible but not currently the target for user events.</dd>
+ <dd>A {{jsxref("String")}} indicating whether or not the session's imagery is visible to the user, and if so, if it's being visible but not currently the target for user events.</dd>
```
Comment (teoli2003) [inline files/en-us/web/api/xrsession/index.html:34]: I wonder if `{{jsxref("String")}}` is the right macro here.

My understanding of WebIDL enum is that values are mapped to a string type (via [toString()](https://tc39.es/ecma262/#sec-tostring)), a litteral without properties or methods, while `{{jsxref("String")}}` links to the `String` object with all its methods and properties.

Am I wrong?

File: files/en-us/web/api/xrsession/visibilitystate/index.html [hunk 2]
```diff
@@ -35,13 +35,29 @@ <h2 id="Syntax">Syntax</h2>
 
 <h3 id="Value">Value</h3>
 
-<p>A {{domxref("DOMString")}} containing one of the values defined in the enumerated type
-  {{domxref("XRVisibilityState")}}; this string indicates whether or not the XR content is
+<p>A {{jsxref("String")}} indicating whether or not the XR content is
```
Comment (teoli2003) [inline files/en-us/web/api/xrsession/visibilitystate/index.html:38]: Ditto.

File: files/en-us/web/api/xrsession/visibilitystate/index.html [hunk 3]
```diff
@@ -35,13 +35,29 @@ <h2 id="Syntax">Syntax</h2>
...
   visible to the user and if it is, whether or not it's currently the primary focus.</p>
 
 <p>The possible values of <code>visibilityState</code> are:</p>
 
-<p>{{page("/en-US/docs/Web/API/XRVisibilityState", "Values")}}</p>
+<dl>
+  <dt><code>hidden</code></dt>
```
Comment (teoli2003) [inline files/en-us/web/api/xrsession/visibilitystate/index.html:44]: Question: shouldn't we put `"hidden"`?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
These are questions, open for discussion.

File: files/en-us/web/api/xrsession/index.html (hunk 1 above)
Comment (Elchi3) [reply files/en-us/web/api/xrsession/index.html:34]: I'm happy to just call say "string" here without any link. I think we discussed this somewhere recently but I couldn't find the issue.

File: files/en-us/web/api/xrsession/visibilitystate/index.html (hunk 3 above)
Comment (Elchi3) [reply files/en-us/web/api/xrsession/visibilitystate/index.html:44]: `hidden` is the name of the thing. Not sure what you mean exactly.

Comment (teoli2003) [reply files/en-us/web/api/xrsession/visibilitystate/index.html:44]: I was thinking about adding the double quotes.

File: files/en-us/web/api/xrsession/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/xrsession/index.html:34]: Yes, I've also seen this discussion. I think just string is fine.

File: files/en-us/web/api/xrsession/visibilitystate/index.html (hunk 3 above)
Comment (teoli2003) [reply files/en-us/web/api/xrsession/visibilitystate/index.html:44]: (Your call here)

Comment (Elchi3) [reply files/en-us/web/api/xrsession/visibilitystate/index.html:44]: Ahh the double quotes, no think we shouldn't. Single quotes or template strings would be fine as well, I think, so let's not assume this.

Comment (teoli2003) [reply files/en-us/web/api/xrsession/visibilitystate/index.html:44]: 👍

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**